js/mage/adminhtml/form.js

```diff
@@ -507,8 +512,17 @@ FormElementDependenceController.prototype = {
      */
     trackChange : function(e, idTo, valuesFrom)
     {
+        let upLevels = this._config.levels_up;
+        let ele;
         if (!$(idTo)) {
-            return;
+            if ($('row_' + idTo)) {
+                idTo = 'row_' + idTo;
+                ele = $(idTo);
+            } else {
+                return;
```

Comment:
```suggestion
            idTo = 'row_' + idTo;
            ele = $(idTo);
            if (!ele) {
                return;
```

Reply:
Maybe we should put the 

```
if (!ele) {
    return;
}
```

beneath all, when we sorted which "element" we want to try to attach to.

What do you think?

Reply:
sure.

Reply:
After consideration I just accepted your change-request as-is.